xbmc/utils/HevcSei.h

```diff
@@ -0,0 +1,54 @@
+/*
+ *  Copyright (C) 2024 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "BitstreamReader.h"
+
+#include <optional>
+#include <vector>
+
+class CHevcSei
+{
+public:
+  CHevcSei() = default;
+  ~CHevcSei() = default;
+
+  uint8_t payload_type{0};
+  size_t payload_size{0};
+
+  // In relation to the input SEI rbsp payload
+  size_t msg_offset{0};
+  size_t payload_offset{0};
+
+  // Parses SEI payload assumed to not have emulation prevention 3 bytes
+  static const std::vector<CHevcSei> ParseSeiRbsp(const uint8_t* buf, const size_t len);
```

Comment:
Please remove the `const` here and the following functions when returning by value, see http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rf-return-const

Reply:
Thanks. Done.